frontends/web/src/routes/market/swap/components/swap-service-selector.module.css

```
@@ -21,6 +24,10 @@
     padding: 0;
 }
 
+.select :global(.react-select__placeholder) {
```

Stylelint (17.10.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)